lib/src/http/managers/sticker_manager.dart

```diff
@@ -0,0 +1,177 @@
+import 'dart:convert';
+
+import 'package:http/http.dart';
+import 'package:nyxx/src/builders/sticker.dart';
+import 'package:nyxx/src/http/managers/manager.dart';
+import 'package:nyxx/src/http/request.dart';
+import 'package:nyxx/src/http/route.dart';
+import 'package:nyxx/src/models/snowflake.dart';
+import 'package:nyxx/src/models/sticker/global_sticker.dart';
+import 'package:nyxx/src/models/sticker/guild_sticker.dart';
+import 'package:nyxx/src/models/sticker/sticker.dart';
+import 'package:nyxx/src/models/sticker/sticker_pack.dart';
+import 'package:nyxx/src/utils/parsing_helpers.dart';
+
+class GuildStickerManager extends Manager<GuildSticker> {
+  final Snowflake guildId;
+
+  GuildStickerManager(super.config, super.client, {required this.guildId}) : super(identifier: "$guildId.stickers");
+
+  @override
+  PartialGuildSticker operator [](Snowflake id) => PartialGuildSticker(id: id, manager: this);
+
+  @override
+  Future<GuildSticker> create(StickerCreateBuilder builder) async {
+    final route = HttpRoute()
+      ..guilds(id: guildId.toString())
+      ..stickers();
+
+    final request = FormDataRequest(route,
+        method: 'POST', formParams: builder.build().cast<String, String>(), files: [MultipartFile.fromBytes('file', builder.file.buildRawData())]);
```

Comment:
Instead of casting, we can just make `StickerCreateBuilder.build` return a `Map<String, String>` since that is a valid override of `Map<String, Object?>`